### PR TITLE
Allow for calling rd_kafka_queue_io_event_enable() from the C++ world

### PR DESCRIPTION
**Pull request by akhi3030**
Hi,

As discussed in, https://github.com/edenhill/librdkafka/issues/617, I've created a pull request.  Awaiting feedback.

Thanks,
Akhi

### CODE REVIEW

File: src-cpp/rdkafkacpp.h [hunk 1]
```diff
@@ -1428,6 +1428,8 @@ class RD_EXPORT Queue {
    */
   virtual int poll (int timeout_ms) = 0;
 
+  virtual void io_event_enable (int fd, const void *payload, size_t size) = 0;
```
Comment (edenhill) [inline src-cpp/rdkafkacpp.h]: Need a doc string here. Copy (and reformat) the one from rdkafka.h

Comment (edenhill) [inline src-cpp/rdkafkacpp.h]: Move this declaration to the end of the class for ABI stability

**Comment by akhi3030** [comment]
Thank you very much for the feedback.  This is my first PR so I hope I've done things correctly.

**Comment by edenhill** [comment]
Thanks for this!

**Pull request merged**